0005-incident_response.md

```diff
@@ -0,0 +1,52 @@
+---
+oip: "0005"
+title: Incident Response
+status: Draft
+type: Operations
+author: Jake Bunce <jake@omse.co>
+created: 2018-11-
+---
+
+# OIP-0005: Incident Response 
+
+## Summary
+
+Outages are common place when running services. Managing these incidents with a defined process helps to restore services faster and creates trust for the users.
+
+## What defines an incident?
+
+An incident is an event that affects normal operation of an OmiseGo service. This can be a service that is used internally or externally, such as an automation service or a Plasma blockchain. Services are measured to determine normal operation through service level indicators (SLIs), from which service level objectives (SLOs) can be applied.
```

Comment:
```suggestion
An incident is an event that affects normal operation of an OmiseGO service. This can be a service that is used internally or externally, such as an automation service or a Plasma blockchain. Services are measured to determine normal operation through service level indicators (SLIs), from which service level objectives (SLOs) can be applied.
```